src/kube-runtime/deploy/delete.sh

```diff
@@ -23,16 +23,24 @@ if kubectl get configmap | grep -q "runtime-exit-spec-configuration"; then
   kubectl delete configmap runtime-exit-spec-configuration || exit $?
 fi
 
-if kubectl get serviceaccount | grep -q "frameworkbarrier-account"; then
+if kubectl get serviceaccount | grep -q "runtime-account"; then
     kubectl delete serviceaccount frameworkbarrier-account || exit $?
```

Comment:
pls do end to end tests

Reply:
Aleady tested in RABC cluster. Runtime can access framework and storage secret

---
In reply to: [352492732](https://github.com/microsoft/pai/pull/3944#discussion_r352492732) [](ancestors = 352492732)

Reply:
I mean you change the account name to `runtime-account` in line 26, but delete `frameworkbarrier-account` in line 27.